### PR TITLE
Destroys Powergaming | Rot Changes - Small balance pass.

### PR DESCRIPTION
**Pull request by Vizoplez**
## About The Pull Request
Rotting of corpses and of people has been way too prevalent for long rounds where not much may be happening. Dying during a fight, and due to some very arbitrary balance changes (weapon smithing quality/ stat adjustments) fights are lasting far longer than normal.

This just ensures that the time to retrieve bodies and revive them (for all sides) is easier. Also technically a necromancer buff, since more corpses.

## Why It's Good For The Game
Rot is extremely oppressive at the moment, as is people powergaming by putting zombie eyes in their skull. Cringe.

### CODE REVIEW

**Comment by Stagrawl** [comment]
Those rot timers are genuinely insane.

**Comment by Stagrawl** [comment]
Zombies are likely going to be reverted to playable. So you can't just gimp them expecting the AI to handle it.

> > Those rot timers are genuinely insane.
> > And are you sure there's not going to be some consequence to fucking eyes as they exist in the heads of zombies?
> 
> You can remove the eyes of/blindfold zombies and they still attack you, iirc they don't even have a sight check.

File: code/datums/components/rotting.dm [hunk 1]
```diff
@@ -113,13 +113,13 @@
 	if(L.stat != DEAD)
 		qdel(src)
 		return
-	if(amount > 10 MINUTES)
+	if(amount > 15 MINUTES) //ROTWOOD EDIT: Things rot slower!
```
Comment (Useroth) [inline code/datums/components/rotting.dm:116]:
```suggestion
	if(amount > 15 MINUTES)
```

File: code/datums/components/rotting.dm [hunk 2]
```diff
@@ -113,13 +113,13 @@
...
 		if(soundloop && soundloop.stopped)
 			soundloop.start()
 		var/turf/open/T = get_turf(L)
 		if(istype(T))
 			T.add_pollutants(/datum/pollutant/rot, 5)
-	if(amount > 20 MINUTES)
+	if(amount > 25 MINUTES) //ROTWOOD EDIT: Though not too slowly!
```
Comment (Useroth) [inline code/datums/components/rotting.dm:122]:
```suggestion
	if(amount > 25 MINUTES)
```

**Pull request merged**